server/app/views/applicant/IneligibleBlockView.java

```diff
@@ -38,14 +38,20 @@ public Content render(
       ReadOnlyApplicantProgramService roApplicantProgramService,
       Optional<String> applicantName,
       Messages messages,
-      long applicantId) {
+      long applicantId,
+      String externalLink) {
     long programId = roApplicantProgramService.getProgramId();
     boolean isTrustedIntermediary = submittingProfile.isTrustedIntermediary();
+    // Use external link if it is present else use the default Program details page
+    String programDetailsLink =
+        externalLink.isEmpty()
+            ? routes.ApplicantProgramsController.view(applicantId, programId).url()
+            : externalLink;
```

Comment:
Here is the fall back if exterlink is empty

Reply:
Thanks!